.flake8

```diff
@@ -10,6 +10,7 @@ per-file-ignores =
     egs/*/ASR/*/optim.py: E501,
     egs/*/ASR/*/scaling.py: E501,
     egs/librispeech/ASR/conv_emformer_transducer_stateless/*.py: E501, E203
+    egs/librispeech/ASR/conv_emformer_transducer_stateless2/*.py: E501, E203
```

Comment:
Maybe you can change it to `egs/*/ASR/conv_emformer_transducer_stateless2/*.py`.

Reply:
Actually, I don't suggest you use this `egs/librispeech/ASR/conv_emformer_transducer_stateless2/*.py: E501, E203` for the all py files in conv_emformer_transducer_stateless2. The better method is focus on some common files or some common code lines, which will not be modified in other datasets' conv_emformer_transducer_stateless2.

Reply:
> Actually, I don't suggest you use this `egs/*/ASR/conv_emformer_transducer_stateless2/*.py: E501, E203` for the all py files in conv_emformer_transducer_stateless2. The better method is focus on some common files or some common code lines, which will not be modified in other datasets' conv_emformer_transducer_stateless2.

Thanks.